test/pdfs/.gitignore

```diff
@@ -168,6 +168,7 @@
 !issue6612.pdf
 !alphatrans.pdf
 !pattern_text_embedded_font.pdf
+!issue9297.pdf
```

Comment:
The test changes in this PR can be removed because we don't have support for testing printing, so this test does not actually test printing at all now.

Reply:
Removed and rebased.